include/winsparkle-version.h

```diff
@@ -31,7 +31,7 @@
  *--------------------------------------------------------------------------*/
 
 #define WIN_SPARKLE_VERSION_MAJOR   0
-#define WIN_SPARKLE_VERSION_MINOR   6
+#define WIN_SPARKLE_VERSION_MINOR   7
```

Comment:
This is unrelated to the rest and shouldn't be included in this commit (or, in fact, this PR at all). Please follow the "one commit per one logical thing" rule of git use.